jsonschema2pojo-integration-tests/src/test/resources/schema/ref/httpRefs.json

```diff
@@ -2,10 +2,10 @@
     "type" : "object",
     "properties" : {
         "address" : {
-            "$ref" : "https://jsonschema2pojo.googlecode.com/git-history/jsonschema2pojo-0.3.4/jsonschema2pojo-integration-tests/src/test/resources/schema/ref/address.json"
```

Comment:
This was failing one of the tests so I replaced it with a working URL to make sure the test passed. Not sure if this is the URL that should be used. Oddly enough this seemed to be passing yesterday.